.travis.yml

```diff
@@ -2,11 +2,10 @@ dist: xenial
 language: ruby
 sudo: false
 rvm:
-  - 2.6
-  - 2.5
+  - 2.7
```

Comment:
The required Ruby version for ActiveRecord 7 is >= 2.7.0 (https://rubygems.org/gems/activerecord/versions/7.0.2.4). So this change makes sense then, correct?

This gem has a "stable-6.0" branch to support development on earlier versions.